pom.xml

```diff
@@ -504,7 +505,7 @@
       <dependency>
         <groupId>org.apache.httpcomponents</groupId>
         <artifactId>httpcore</artifactId>
-        <version>${commons.httpclient.version}</version>
```

Comment:
See https://mvnrepository.com/artifact/org.apache.httpcomponents/httpclient/4.5.2 which depends of httpcore 4.4.4